etc/adminhtml/system/adyen_capture_settings.xml

```diff
@@ -24,17 +24,11 @@
             <source_model>Adyen\Payment\Model\Config\Source\CaptureMode</source_model>
             <config_path>payment/adyen_abstract/capture_mode</config_path>
         </field>
-        <field id="capture_on_shipment" translate="label" type="select" sortOrder="20" showInDefault="1" showInWebsite="1" showInStore="1">
-            <label>Capture on shipment for OpenInvoice Methods (Klarna, Ratepay and Afterpay)</label>
-            <tooltip>If you enable this function it will do a capture request to Adyen if you do a shipment for openinvoice payment methods.</tooltip>
-            <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
-            <config_path>payment/adyen_abstract/capture_on_shipment</config_path>
-        </field>
-        <field id="auto_capture_openinvoice" translate="label" type="select" sortOrder="30" showInDefault="1" showInWebsite="1" showInStore="1">
-            <label>Use auto-capture for OpenInvoice payments</label>
-            <tooltip>Applicable for Klarna, Afterpay Touch, Ratepay, FacilyPay/Oney, Affirm, Clearpay, Zip and PayBright. By default, Open Invoice methods are set to manual capture. If you want auto capture you need to contact magento@adyen.com. After approval has been given you can set this option to 'Yes'.</tooltip>
-            <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
-            <config_path>payment/adyen_abstract/auto_capture_openinvoice</config_path>
+        <field id="capture_for_openinvoice" translate="label" type="select" sortOrder="20" showInDefault="1" showInWebsite="1" showInStore="1">
+            <label>Capture for OpenInvoice Methods</label>
+            <tooltip>Applicable for Klarna, Afterpay Touch, Ratepay, FacilyPay/Oney, Affirm, Clearpay, Zip and PayBright. By default, Open Invoice methods are set to manual capture. If you want auto capture you need to contact magento@adyen.com. After approval has been given you can set this option to 'Immediate'. If you want to do a capture request to Adyen if you do a shipment then set it to 'On Shipment'.</tooltip>
+            <source_model>Adyen\Payment\Model\Config\Source\OpenInvoiceCaptureMode</source_model>
+            <config_path>payment/adyen_abstract/capture_for_openinvoice</config_path>
```

Comment:
Could you please revisit label and tooltip together with our UX writer? Maybe we can simplify this text.